demisto_sdk/commands/content_graph/strict_objects/incident_field.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any, List, Optional
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import IncidentFieldType
+from demisto_sdk.commands.content_graph.strict_objects.base_strict_model import (
+    NAME_DYNAMIC_MODEL,
+    BaseStrictModel,
+)
+from demisto_sdk.commands.content_graph.strict_objects.indicator_field import (
+    StrictIndicatorField,
+)
+
+
+class Aliases(BaseStrictModel):
+    __base__ = NAME_DYNAMIC_MODEL
```

Comment:
```suggestion
class Aliases(NAME_DYNAMIC_MODEL):
```
Will this change work? It's less confusing and ~does~ should do the same